Projections/Test/Test.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
     <Platforms>x64;x86</Platforms>
-  </PropertyGroup>
+   </PropertyGroup>

```

Comment:
Did we not set it here?

Reply:
I didn't originally because I didn't think it would make much sense to enable it in test projects. But this morning I had a new thought that it wouldn't be bad to add it in these places. Not sure how this whitespace change got committed, if that's what you're asking.